e2e/manifests/staging.toml

```diff
@@ -1,5 +1,5 @@
 network = "staging"
-public_chains = ["op_sepolia"]
+public_chains = ["op_sepolia", "base_sepolia"]
```

Comment:
let's just add it to omega, no need for staging to include it imo

Reply:
we don't include arb sepolia

Reply:
Done!